sdk/core/core-amqp/src/retry.ts

```diff
@@ -256,3 +257,14 @@ export async function retry<T>(config: RetryConfig<T>): Promise<T> {
     throw lastError;
   }
 }
+
+export function getRetryAttemptTimeoutInMs(retryOptions: RetryOptions | undefined): number {
```

Comment:
Moved this from event-hubs to core-amqp to reuse it for service-bus.
Let me know if this should be duplicated instead(to get away with version updates).

Reply:
Reverted and keeping the duplications so that we don't have to release core-amqp for this change.